helper-cli/src/main/kotlin/commands/dev/RewriteTestAssetsCommand.kt

```diff
@@ -0,0 +1,209 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands.dev
+
+import com.fasterxml.jackson.core.JacksonException
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.ObjectNode
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+
+import java.io.File
+
+import kotlin.reflect.KClass
+
+import org.ossreviewtoolkit.model.AnalyzerResult
+import org.ossreviewtoolkit.model.AnalyzerRun
+import org.ossreviewtoolkit.model.EvaluatorRun
+import org.ossreviewtoolkit.model.FileFormat
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.ProjectAnalyzerResult
+import org.ossreviewtoolkit.model.ScanResult
+import org.ossreviewtoolkit.model.ScannerRun
+import org.ossreviewtoolkit.utils.common.expandTilde
+
+internal class RewriteTestAssetsCommand : CliktCommand(
+    help = "Searches all test asset dirs in the given ORT sources directory for '.json', '.yml' and '.yaml' files " +
+            "and tries to de-serialize and serialize the file."
+) {
+    private val ortSourcesDir by option(
+        "--ort-sources-dir", "-i",
+        help = "The directory containing the source code of ORT."
+    ).convert { it.expandTilde() }
+        .file(mustExist = true, canBeFile = false, canBeDir = true, mustBeWritable = false, mustBeReadable = true)
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    override fun run() {
+        val candidateFiles = findCandidateFiles(ortSourcesDir)
+        println("Found ${candidateFiles.size} candidate files.\n")
+
+        val candidateFilesByType = patchAll(candidateFiles)
+
+        candidateFilesByType.forEach { (type, files) ->
+            files.forEach { file ->
+                println("[$type] ${file.absolutePath}")
+            }
+        }
+
+        println("\nSummary\n")
+
+        candidateFilesByType.forEach { (type, files) ->
+            println("$type: ${files.size}")
+        }
+    }
+}
+
+private val REPLACE_PATTERN_REPLACEMENTS = listOf(
+    "<REPLACE_JAVA>" to "33445501",
+    "<REPLACE_OS>" to "33445502",
```

Comment:
Nit: Should this string type also use a non-numeric fake value?

Reply:
See above.